Formula/libetonyek.rb

```diff
@@ -28,6 +28,12 @@ class Libetonyek < Formula
   resource "liblangtag" do
     url "https://bitbucket.org/tagoh/liblangtag/downloads/liblangtag-0.6.3.tar.bz2"
     sha256 "1f12a20a02ec3a8d22e54dedb8b683a43c9c160bda1ba337bf1060607ae733bd"
+
+    # Fix -flat_namespace being used on Big Sur and later.
+    patch do
+      url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
+      sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
+    end
```

Comment:
You could just update to 0.6.4 where this issue is fixed.

Reply:
We should definitely update 0.6.4 in that case, but wouldn't updating the version without updating the bottles break the bottle manifest?  I've messed this up before by merging version updates without publishing the bottles and I'm pretty sure the solution ended up being that we had to build new bottles anyway.

Reply:
This is a resource update - not a formula update.

Reply:
That makes sense, thanks for the explanation!